### PR TITLE
Clarify usage of additional options for path qualifier

### PR DESCRIPTION
**Pull request by mribeirodantas**
The idea for this pull request came to be due to [this discussion](https://nextflow.slack.com/archives/C02T98A23U7/p1668691328381029) on Nextflow Slack.

### CODE REVIEW

File: docs/process.rst [hunk 1]
```diff
@@ -1009,6 +1009,33 @@ Name                Description
...
+      '''
+      echo 'another new line' >> result.txt
+      '''
+    }
+
+Another example with multiple output qualifiers::
+
+    process foo {
+      output:
+      path('last_result.txt'), path('result.txt', hidden: true)
```
Comment (pditommaso) [inline docs/process.rst:1031]:
```suggestion
      path('result.txt', hidden: true)
```

guess you meant this ☝️

Comment (mribeirodantas) [reply docs/process.rst:1031]: Oh, I forgot the `tuple` at the beginning 😅

Comment (mribeirodantas) [inline docs/process.rst:1031]:
```suggestion
      tuple path('last_result.txt'), path('result.txt', hidden: true)
```

**Comment by mribeirodantas** [comment]
👀 😬

**Comment by mribeirodantas** [comment]
LGTM, @pditommaso ? 😄

**Review by pditommaso — APPROVED** [review]
Excellent

**Pull request merged**